ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhoai-operator-interop__v4.15-production.yaml

```diff
@@ -0,0 +1,173 @@
+base_images:
```

Comment:
do we have `4.15` in production?

Reply:
@rnetser yes I did created 4.15 OSD cluster in prod

Reply:
but this is `rosa`

Reply:
removed ROSA configs from both stage and prod, not able to create for 4.15 nightly build